packages/drivers/routerlicious-driver/src/routerliciousResolvedUrl.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IResolvedUrl } from "@fluidframework/driver-definitions";
+
+/**
+ * Routerlicious extends the resolved url with additional properties to control Routerlicious-specific behaviors.
```

Comment:
Nit: Might be worth a `@remarks` comment noting the existence of `isRouterliciousResolvedUrl` (for enhanced discoverability).

Reply:
Done in latest - I used `@privateRemarks` since I'm not exporting that function from the package (intentionally - only the driver needs to introspect the resolvedUrl type (at least for now)) and unexported links makes `@remarks` unhappy.

Reply:
I didn't realize API-Extractor was smart enough to skip validation of links in `@privateRemarks` blocks. That's awesome and great to know!